hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/router/FederatedNamespaceIds.java

```diff
@@ -0,0 +1,78 @@
+/**
```

Comment:
I think this functionality should be merged into RouterStateIdContext, which serves the same purpose and wraps the instance of this class in the Router.

Reply:
Fixed.